Puppetfile

```diff
@@ -61,7 +61,8 @@ mod 'KyleAnderson/etcd',
 mod 'puppetlabs/lvm',
   :git => 'git://github.com/alphagov/puppetlabs-lvm.git'
 mod 'puppetlabs/rabbitmq',
-  :git => 'git://github.com/puppetlabs/puppetlabs-rabbitmq.git'
+  :git => 'git://github.com/puppetlabs/puppetlabs-rabbitmq.git',
+  :ref => '5.3.1'
```

Comment:
Since this bumps us to a released version we should probably stop using the git version and just use the [forge version](https://forge.puppet.com/puppetlabs/rabbitmq).